pdal/SpatialReference.hpp

```diff
@@ -182,12 +182,15 @@ class PDAL_DLL SpatialReference
 
 private:
     std::string m_wkt;
+    mutable std::string m_wkt2;
```

Comment:
I don't understand why this is `mutable` and *m_wkt* isn't.

Reply:
> I don't understand why this is `mutable` and _m_wkt_ isn't.

m_wkt is always set in the set() method, whereas m_wkt2 is only set in set() if the WKT is WKT2. Otherwise it is computed on demand in getWKT2() which is const, hence the mutable.
The alternative would be that I always compute m_wkt2, and thus getWKT2() would just return it

Reply:
I think I would rather always compute it in set() so it is clear what's happening, instead of having the scary `mutable` without the comment to tell us why it is mutable.

Reply:
> I think I would rather always compute it in set() so it is clear what's happening, instead of having the scary `mutable` without the comment to tell us why it is mutable.

done